### DISCUSSION TITLE
Wrong value when time=0 in output file

### DISCUSSION THREAD

**Discussion by hokonkim** [idaholab/moose#18165] [Q&A General] [answered]
![image](https://user-images.githubusercontent.com/46009457/123330139-a2588980-d50b-11eb-9325-bd1ba5881728.png)

![image](https://user-images.githubusercontent.com/46009457/123330262-ca47ed00-d50b-11eb-9110-3aedbc44ae41.png)

Above pictures are parts of my code.

What I expected was "phi_Ni = -1.03665 in output file" when time = 0.

But when time=0, phi_Ni = 0 was shown.

![image](https://user-images.githubusercontent.com/46009457/123331020-b94bab80-d50c-11eb-8e72-e10acf46f2ae.png)

What do I miss at here?

Thanks!

**Comment by GiudGiud** [comment, marked as answer]
Hello

the postprocessor is not being executed at the 0th timestep. 
You need to add execute_on = 'INITIAL timestep_end' to your input file for this postprocessor

Guillaume

**Reply by hokonkim** [reply]
Thanks, it works!
